example.yml

```diff
@@ -164,49 +164,31 @@ git:
 
   # Only pull from specific projects / organizations / groups.
   # Bitbucket Server: use project keys, not names. Comment this out to pull from all projects.
-  # Bitbucket Cloud (required): use your organization name, as it appears in your Bitbucket URLs
+  # Bitbucket Cloud (required): use your workspace name, as it appears in your Bitbucket URLs
   # GitHub (required): use your organization name, as it appears in your GitHub URLs (e.g.: MyOrg)
   # GitLab (required): use the top-level GitLab "Group ID" (e.g.: 123)
   # Azure Devops (required): use your organization name, which contains your projects
   include_projects:
-      - PROJ1
+      - PROJ1  # or, 123 for Gitlab
 
-  # Uncomment this to pull from all but specific projects (not supported for GitHub or Bitbucket Cloud).
+  # Uncomment this to pull from all but specific projects (not supported for GitHub, GitLab, BitbucketCloud, or Azure Devops).
   # Bitbucket Server: use project keys, not names (e.g.: PROJ1)
   # GitLab: use the top-level GitLab "Group ID" (e.g.: 123)
   #  exclude_projects:
-  #    - PROJ1
-
-  # Bitbucket cloud only: the include_projects filter above maps to
-  # top-level organization for bitbucket cloud. If you've grouped your
-  # repos into Bitbucket Cloud projects, use these filters instead to
-  # pull only from repos in particular projects.  Specify projects by
-  # key (e.g.: PROJ1)
-  # include_bitbucket_cloud_projects:
-  #     - PROJ1
-  #
-  # exclude_bitbucket_cloud_projects:
-  #     - PROJ1
```

Comment:
These two fields, `include_bitbucket_cloud_projects` and `exclude_bitbucket_cloud_projects` have been deprecated for over 4 years now. Users have to specifically state what projects they want to crawl over for bitbucket cloud (via `include_projects` above), so this is redundant and confusing

Reply:
Internally marked as legacy code [here](https://github.com/Jellyfish-AI/jf_agent/blob/ca0617e6912e15a6688ecb62f6cf7288ca0a2e14/jf_agent/config_file_reader.py#L645) and [here](https://github.com/Jellyfish-AI/jf_agent/blob/ca0617e6912e15a6688ecb62f6cf7288ca0a2e14/jf_agent/config_file_reader.py#L43)